src/js/API.js

```diff
@@ -0,0 +1,38 @@
+import { Game } from './game.js';
+
+const idApi = 'J8ulCJbuTzxWT8xwonUg';
+const MyURL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${idApi}/scores/`;
+const game = new Game();
+
+async function getData(url, options = {}) {
+  const response = await fetch(url, options)
+    .then((response) => response.json())
+    .then((data) => data.result)
+    .catch((error) => console.log(error));
+  return response;
+}
+
+function passOptions(objOption) {
```

Comment:
- [ ] Please make use of arrow function instead of the function keyword. For more information on how the ES6 arrow function syntax works, see this [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

Reply:
Thanks for your feedback @Ghiftee, It is now solved on commit f0d37df.